### PR TITLE
NORA-B1XX boards: Make them compatible with NCS 2.6

### PR DESCRIPTION
**Pull request by jakkra**
Needed to build for nRF Connect SDK 2.6 as hci_rpmsg has been renamed to hci_ipc

### CODE REVIEW

File: zephyr/boards/arm/archive/ubx_evknorab10_nrf5340_ncs220/Kconfig.defconfig
Comment (perssonmagnus) [inline zephyr/boards/arm/archive/ubx_evknorab10_nrf5340_ncs220/Kconfig.defconfig:1]: Should we change the files in the 'archive' folder?

**Review by perssonmagnus — COMMENTED** [review]
Not sure if we should change the files in archive, looks good to me otherwise.
Maybe you could update https://github.com/u-blox/u-blox-sho-OpenCPU/blob/master/README.md as well to say 2.6?

File: zephyr/boards/arm/archive/ubx_evknorab10_nrf5340_ncs220/Kconfig.defconfig
Comment (jakkra) [reply zephyr/boards/arm/archive/ubx_evknorab10_nrf5340_ncs220/Kconfig.defconfig:1]: > Should we change the files in the 'archive' folder?

Sorry, no, I did find and replace... Will fix.

**Comment by jakkra** [comment]
I have only tested the MINI-Noras hence changed only that in README.
Do note this breaks backwards compatability with NCS2.5. Not sure how that is handled here, maybe a branching strategy is needed for that with tags for different NCS versions etc.

**Comment by perssonmagnus** [comment]
Looks good to me, but I'll prefer if @brec-u-blox  do final approval.

**Review by ghost — APPROVED** [review]

**Pull request merged**